articlequality/utilities/extract_scores.py

```diff
@@ -257,3 +267,8 @@ def _threshold_scores(dump, path):
                            error_score)
 
     return _threshold_scores
+
+
+def _parse_class_weight_option(class_weight_option_string):
+    (key, weight_string) = class_weight_option_string.split('=')
+    return (key.strip('"'), int(weight_string))
```

Comment:
It would be safer to use something like json for interpreting the "key" since there are non-string types that can be used as output classes for a model.

Reply:
Here's an example of using json to process these arguments: https://github.com/wikimedia/revscoring/blob/c472a74e47007a262d75af53b761dba41c6e7aac/revscoring/utilities/util.py#L69

Essentially, you can do something like this: 

```
def _parse_class_weight_option(class_weight_option_string):
    class_str, weight_string = class_weight_option_string.split('=', 1)
    return json.loads(class_str), json.loads(weight_string)
```

Reply:
Done, I think. Does it make sense to set the weight anything other than an integer?

Reply:
Yes!  In euwiki, they have used floats in the past.